testproject/Assets/Tests/Runtime/MultiprocessRuntime/BaseMultiprocessTests.cs

```diff
@@ -109,18 +109,36 @@ public virtual IEnumerator Setup()
             MultiProcessLog("m_HasSceneLoaded");
             var startTime = Time.time;
 
+            MultiProcessLog($"Active Worker Count is {MultiprocessOrchestration.ActiveWorkerCount()} and connected client count is {NetworkManager.Singleton.ConnectedClients.Count}");
+            if (MultiprocessOrchestration.ActiveWorkerCount() + 1 < NetworkManager.Singleton.ConnectedClients.Count)
+            {
+                MultiProcessLog("Is this a bad state?");
+            }
+
             // Moved this out of OnSceneLoaded as OnSceneLoaded is a callback from the SceneManager and just wanted to avoid creating
             // processes from within the same callstack/context as the SceneManager.  This will instantiate up to the WorkerCount and
             // then any subsequent calls to Setup if there are already workers it will skip this step
-            if (MultiprocessOrchestration.ActiveWorkerCount() < WorkerCount)
+            if (NetworkManager.Singleton.ConnectedClients.Count - 1 < WorkerCount)
             {
-                var numProcessesToCreate = WorkerCount - MultiprocessOrchestration.ActiveWorkerCount();
-                for (int i = 0; i < numProcessesToCreate; i++)
+                var timeOutTime2 = Time.realtimeSinceStartup + TestCoordinator.MaxWaitTimeoutSec / 3;
```

Comment:
/ 3? Where's that coming from?
If it's because MaxWaitTimeoutSec is too long, why not just have a separate one for this case?

Reply:
Actually, now that I'm looking at it, I was intending to do some extra processing to start a new worker if the previous worker somehow didn't start up but that's a really bad idea and would hide a legit failure in the case that the worker failed to start. I'll revert this change.

Reply:
addressed in https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1195/commits/4d26c46c0d7b955b296b3162b28c995fdd2c45b3